supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -220,6 +242,40 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 *See also*:
 
 // RHEL: Added "In Red Hat Enterprise Linux,"
+
+[[block-device]]
+==== image:images/caution.png[with caution] block device (noun)
+*Description*: A _block device_ is a physical storage device that supports reading and writing data in groups (blocks) of bytes in any order. Do not confuse this term with _block storage_ (cloud storage) or _block volume_ (OpenShift Virtualization).
+
+*Use it*: with caution
+
+[.vale-ignore]
+*Incorrect forms*: block storage, block volume
+
+*See also*:
```

Comment:
Add "block storage and block volume" to "See also"?

Reply:
Added xrefs to all 3 entries to each other